src/blockchain_utilities/blockchain_export.cpp

```diff
@@ -72,11 +71,9 @@ int main(int argc, char* argv[])
   };
   const command_line::arg_descriptor<bool> arg_blocks_dat = {"blocksdat", "Output in blocks.dat format", blocks_dat};
 
-
-  command_line::add_arg(desc_cmd_sett, cryptonote::arg_data_dir, default_data_path.string());
-  command_line::add_arg(desc_cmd_sett, cryptonote::arg_testnet_data_dir, default_testnet_data_path.string());
-  command_line::add_arg(desc_cmd_sett, arg_output_file);
   command_line::add_arg(desc_cmd_sett, cryptonote::arg_testnet_on);
+  command_line::add_arg(desc_cmd_sett, cryptonote::arg_data_dir);
+  command_line::add_arg(desc_cmd_sett, arg_output_file);
```

Comment:
I don't understand that one. git seems to find a difference in that last line, but I can't see one. Is there a difference in your local copy ?

Reply:
Sorry, missed that. This is because `arg_testnet_on` line was moved up, which probably seemed logical to me at that moment, knowing that `arg_data_dir` depends on it. Should I fix this, or leave it as it is, since adding another commit is redundant and forcing the branch would break your approval?

Reply:
Ah, it seems obvious in retrospect.Silly me. In general I don't want stuff moved around unless it needs to, since that causes confusing diffs (such as this one), but for one line in modified code, it's fine.

Reply:
IMO the ordering doesn't need to be changed, since it has nothing to do with the dependency relation which is defined in cryptonote_core.cpp. This ordering change seems a bit confusing.

Reply:
Reverted it in 93bd692.